examples/aides_logement/tests/tests_calcul_al_accession_propriete.catala_fr

```diff
@@ -75,24 +75,26 @@ champ d'application Exemple2:
 ```catala-test-inline
 $ catala Interpret -s Exemple1
 [RESULT] Computation successful! Results:
-[RESULT] montant = 96.48 €
+[RESULT] montant = ¤96.48
 ```
 
 ```catala-test-inline
 $ catala Interpret_Lcalc -s Exemple1 --avoid_exceptions
 [RESULT] Computation successful! Results:
-[RESULT] montant = ESome 96.48 €
+[RESULT] montant = ESome ¤96.48
 ```
 
 ```catala-test-inline
 $ catala Interpret -s Exemple2
 [RESULT] Computation successful! Results:
-[RESULT] montant = 85.00 €
+[RESULT] montant = ¤85.00
 ```
 
 ```catala-test-inline
 $ catala Interpret -s Exemple2 --avoid_exceptions
-[RESULT] Computation successful! Results:
-[RESULT] montant = 85.00 €
+catala: unknown option '--avoid_exceptions'.
```

Comment:
This one should not fail like this

Reply:
Ah yes missed that one, the option must be removed (it makes no sense with `--interpret`)